libs/portals/admin/ids-admin/src/screens/Tenant/Tenant.tsx

```diff
@@ -3,12 +3,17 @@ import { idsAdminNav } from '../../lib/navigation'
 import Layout from '../../components/Layout/Layout'
 import { useLocale } from '@island.is/localization'
 import { m } from '../../lib/messages'
+import { useState } from 'react'
 
 const Tenant = () => {
   const { formatMessage } = useLocale()
+  const [title, setTitle] = useState<string>(formatMessage(m.idsAdmin))
   return (
-    <Layout navTitle={formatMessage(m.idsAdmin)} navItems={idsAdminNav}>
-      <Outlet />
+    <Layout
+      navTitle={title ?? formatMessage(m.idsAdmin)}
+      navItems={idsAdminNav}
+    >
+      <Outlet context={{ setTitle }} />
```

Comment:
This component should be able to get the tenant info by itself with `useRouteLoaderData(tenantLoaderId)`, so it works for all routes, not just Clients.tsx.

Reply:
That is correct!